src/components/HelpModal.js

```diff
@@ -0,0 +1,22 @@
+import ReactModal from 'react-modal'
+import '../css/HelpModal.css'
+
+function HelpModal({ isOpen, close}) {
+    return (
+        <ReactModal className="modal" isOpen={isOpen} parentSelector={() => document.querySelector('.Game')} contentLabel="Instructions">
+            <div className='help'>
+                <button className="close-btn" onClick={close}>🆇</button>
+                <h3>How to Play:</h3>
+                <p className="instructions">1. Select <span>"Set Game Board!"</span> at the bottom of the screen. You'll see country names appear in the game board squares.</p>
+                <p className="instructions">2. Select <span>"Get Coordinates"</span> above the game board. You'll see a set of coordinates appear.</p>
+                <p className="instructions">3. Use the map to <span>find the country</span> at the given coordinates.</p>
+                <p className="instructions">4. If that country appears on your game board, <span>select that square</span> and get the next set of coordinates.</p>
```

Comment:
Again -- "click that square," maybe?

Do we want to add an aside -- "(Hint: Not every country will be on your bingo board! Look carefully!)" or something to that effect?

Reply:
Implemented